app/login/index.js

```diff
@@ -0,0 +1,16 @@
+const stickyNavBar = () => {
+	const navbar = document.getElementById('navbar');
+	const offsetPositionOfNavBar = navbar.offsetTop;
+
+	const toggleStickyClass = () => {
+		if (window.pageYOffset >= offsetPositionOfNavBar) {
+			navbar.classList.add('sticky');
+		} else {
+			navbar.classList.remove('sticky');
+		}
+	};
+
+	window.onscroll = () => toggleStickyClass();
+};
+
+stickyNavBar();
```

Comment:
I think I might prefer to just have a navbar script that both the homepage and the login page load. (Of course this type of problem doesn't exist with a compilation pipeline and/or react, but such is vanilla JS!)